template/tsconfig/cypress-ct/tsconfig.cypress-ct.json

```diff
@@ -1,15 +1,17 @@
 {
   "extends": "./tsconfig.app.json",
   "include": [
-    "env.d.ts",
-    "src/**/*",
-    "src/**/*.vue",
+    "src/**/__tests__/*",
     "cypress/support/component.*",
     "cypress/support/commands.ts"
   ],
   "exclude": [],
   "compilerOptions": {
     "composite": true,
+    "outDir": "./node_modules/.tmp/cypress-ct",
```

Comment:
It's not strictly necessary to have the declarations output for anything but the tsconfig.app.json right now, but I think it makes sense to make them all work the same way.

Reply:
> It's not strictly necessary to have the declarations output for anything but the tsconfig.app.json right now

Note that this will change in TypeScript 5.6, see https://github.com/microsoft/TypeScript/issues/32651#issuecomment-2260378908